training/src/app/modules/chuck-norris/components/joke-generator/joke-generator.component.scss

```diff
@@ -0,0 +1,92 @@
+@import "../../../../../styles";
+
```

Comment:
use pixel instead of percentage for elements, unless for example its an element that u want 80% of the screen for example